test/cmd/apply.sh

```diff
@@ -223,6 +223,21 @@ __EOF__
   # cleanup
   kubectl delete -f hack/testdata/service-revision2.yaml "${kube_flags[@]}"
 
+  ## kubectl apply -k somedir
+  kubectl apply -k hack/testdata/kustomize
```

Comment:
lets add a test for every command that supports this if possible

Reply:
How about adding tests to the existing test files for `apply`, `diff`, `create`, `delete`, `get`?

Reply:
+1